cynthion/python/examples/tutorials/gateware-blinky.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+#
+# This file is part of Cynthion.
+#
+# Copyright (c) 2024 Great Scott Gadgets <info@greatscottgadgets.com>
+# SPDX-License-Identifier: BSD-3-Clause
+
+from amaranth import *
+
+class Top(Elaboratable):
+    def elaborate(self, platform):
+        m = Module()
+
+        leds: Signal(6) = Cat(platform.request("led", n).o for n in range(0, 6))
+
+        half_freq: int    = int(60e6 // 2)
+        timer: Signal(25) = Signal(range(half_freq + 1))
+
+        with m.If(timer == half_freq):
```

Comment:
I think the correct thing would be to check for `timer == half_freq - 1`. This is adding an extra cycle so the blinking period is not 500 ms (but really close!). For that reason, `timer` could just be `Signal(range(half_freq))`.

Reply:
Fixed!

Reply:
https://github.com/greatscottgadgets/cynthion/pull/197/commits/86597d9a966bb12777e6499054ed9f14597835de